File: projects/m1/004-units-of-time/solutions/GabrieleGabellone/units_of_time.py
#UNITS-OF-TIME
#Create a program that reads a duration from the user as a number of days, hours, minutes, and seconds. 
# Compute and display the total number of seconds represented by this duration.

d = int (input("Enter the number of days: "))
h = int (input("Enter the number of hours: "))
m = int (input("Enter the number of minutes: "))
s = int (input("Enter the number of seconds: "))
total_seconds = (d * 86400) + (h * 3600) + (m * 60) + s
print ("In total it is", total_seconds, "seconds")

File: projects/m1/004-units-of-time/solutions/vittorioMenicanti/unitsOftime.js


// units of time

const secondsPerMinute = 60;
const secondsPerHour = 3600;
const secondsPerDay = 86400;

const days = prompt(`Please enter the number of Days Here:`);
const hours = prompt(`Please enter the number of Hours Here:`);
const minutes = prompt(`Please enter the number of Minutes Here:`);
const seconds = prompt(`Please enter the number of Seconds Here:`);

const totalSeconds = days * 86400 + hours * 3600 + minutes * 60 + seconds;

alert(`The total number of seconds is: ${totalSeconds}`);